staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -742,7 +742,10 @@ func (s *store) getStateFromObject(obj runtime.Object) (*objState, error) {
 	if err != nil {
 		return nil, err
 	}
-	s.versioner.UpdateObject(state.obj, uint64(rv))
+	err = s.versioner.UpdateObject(state.obj, uint64(rv))
+	if err != nil {
+		klog.Errorf("failed to update object: %v", err)
```

Comment:
failed to update object version

Reply:
done. thx.

Reply:
```go
if err := s.versioner.UpdateObject(state.obj, uint64(rv)); err != nil {
...
}
```

Reply:
@answer1991 thx. I have already moved line 745 into the `if` clause. I suppose my code above may be outdated.